src/VersionsCollection.php

```diff
@@ -28,11 +28,23 @@ public function __construct(Version ...$versions)
         $this->versions = $versions;
     }
 
+    public static function fromStrings(string ...$versionStrings) : VersionsCollection
```

Comment:
Wouldn't add this, tbh. New runtime exceptions rather than just a typed ctor

Reply:
The only reason why I wanted to restore this feature was that I saw a lot of use cases in Roave/BackwardCompatibilityCheck where it might come handy, though those were mostly in tests. But I agree that type safety should not be sacrificed here. Removing it.